webapp/channels/src/components/root/root.test.tsx

```diff
@@ -30,24 +28,23 @@ jest.mock('rudder-sdk-js', () => ({
     ready: jest.fn((callback) => callback()),
     track: jest.fn(),
 }));
-
 jest.mock('actions/telemetry_actions');
-
-jest.mock('actions/global_actions', () => ({
-    redirectUserToDefaultTeam: jest.fn(),
-}));
-
+jest.mock('components/announcement_bar', () => () => <div/>);
```

Comment:
Why do these need to be mocked now?

Reply:
Ahh this was brought in from another PR. Reverted